functions/nvm.fish

```diff
@@ -114,7 +116,7 @@ function nvm --description "Node version manager"
                 end
 
                 if ! command curl -q $silent --progress-bar --location $url |
-                        command tar --extract --gzip --directory $nvm_data/$ver 2>/dev/null
+                        command $tarcmd --extract --gzip --directory $nvm_data/$ver 2>/dev/null
```

Comment:
Do you happen to remember which version of Fish started supporting this way of running a command?

Reply:
Are you asking about the `command` builtin or using variable expansion for the program name? I believe both have been present since the first release of Fish.